terraform/modules/monitoring/topic.tf

```diff
@@ -0,0 +1,5 @@
+#tfsec:ignore:aws-sns-enable-topic-encryption:encryption at rest is overkill for the type of data we will be sending to the topic
+resource "aws_sns_topic" "this" {
```

Comment:
AFAIK there will only be one SNS topic which we use for everything. Happy to use a disambiguating name now though if that's not (or might not) be the case.

Reply:
A single topic feels right to me (we're only using this for alarms / notifications to slack)